back-end/models/platform.py

```diff
@@ -6,5 +6,11 @@ class Platform(CamelModel):
     id: int
     name: str
     
+    def __hash__(self):
+        return hash((self.id, self.name))
+
+    def __eq__(self, other):
+        return isinstance(other, Platform) and self.id == other.id and self.name == other.name
```

Comment:
[Should] So sánh bằng cách override __eq__ khiến người đọc khó biết được cách so sánh.
Anh nghĩ em nên so sánh trực tiếp a.id == b.id and a.name == b.name trong hầu hết trường hợp. Nếu dùng các hàm như sort thì có thể dùng kwarg keys.